### PR TITLE
feat(telemetry): add cache hits ratio

### PR DESCRIPTION
**Pull request by layershifter**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

This PR improves `telemetry` object an provides more data: cache hits and differentiates renders from components' instances.

Also exposes `getFluentTelemetry()` to get telemetry data in Fluent doc site:

image
 

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/fluentui/docs/src/app.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import Routes from './routes';
 
 // Experimental dev-time accessibility attributes integrity validation.
 import { setup } from '@fluentui/ability-attributes';
+import { Telemetry } from '../../react-bindings/src';
```
Comment (mnajdova) [inline packages/fluentui/docs/src/app.tsx:11]:
```suggestion
import { Telemetry } from '@fluentui/react-bindings';
```
?

File: packages/fluentui/CHANGELOG.md [hunk 2]
```diff
@@ -81,9 +81,11 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Use `compose()` in `ToolbarRadioGroup` @layershifter ([#13322](https://github.com/microsoft/fluentui/pull/13322))
 - Use `compose()` in `Toolbar` @mnajdova ([#13341](https://github.com/microsoft/fluentui/pull/13341))
 - Use `compose()` in `ToolbarItem` @mnajdova ([#13321](https://github.com/microsoft/fluentui/pull/13321))
+- Add cache hits to telemetry data & differentiate initial render @layershifter ([#13346](https://github.com/microsoft/fluentui/pull/13346))
 - Use `compose()` in `ToolbarMenuItem` @mnajdova ([#13350](https://github.com/microsoft/fluentui/pull/13350))
 - Use `compose()` in `ToolbarCustomItem` @mnajdova ([#13319](https://github.com/microsoft/fluentui/pull/13319))
 
+
```
Comment (layershifter) [inline packages/fluentui/CHANGELOG.md:88]:
```suggestion
```

**Comment by mnajdova** [comment]
I think that the telemetry data showed us a problem with composed component's caching, or the calculations are wrong.
image
Basically we don't have any cache hits for composed components. I would suggest looking into this before merging, at least to ensure that the data is correct, and we can investigate if there is any issue with the caching.. Other than that, the changes look good 👍

File: packages/fluentui/docs/src/app.tsx [hunk 3]
```diff
@@ -20,6 +21,21 @@ const themes = {
   teamsHighContrastTheme,
 };
 
+const TelemetryGuard: React.FC<{
```
Comment (mnajdova) [inline packages/fluentui/docs/src/app.tsx:24]: 👍

**Pull request merged**